packages/core/src/agent/AgentConfig.ts

```diff
@@ -71,6 +71,10 @@ export class AgentConfig {
     return this.initConfig.autoUpdateStorageOnStartup ?? false
   }
 
+  public get backupBeforeStorageUpdate() {
```

Comment:
One thing to consider though that not creating a backup may be seen as advanced and you should use the update assistant in this case? But if it's useful to add this to the default agent startup, then I'm ok with it 👍